src/Scene/BrowseTree.js

```diff
@@ -173,15 +173,16 @@ BrowseTree.prototype.updateLayer = function updateLayer(layer, camera) {
 
     var root = layer.children[0];
 
-    for (var c = 0; c < root.children.length; c++) {
-        var node = root.children[c];
+    for (let c = 0; c < root.children.length; c++) {
+        const node = root.children[c];
 
         var cRTC = function getCRtcFn() {
```

Comment:
Oh I get it! It's the `const` declaration, that correctly scopes the variable and avoid the coding error described in the doc where your closure doesn't contain what you believe it does.